setup.py

```diff
@@ -51,6 +51,7 @@
     "WTForms<3.0.0,>=2.3.3",
     # invenio-celery conflict fix
     "celery<5.2,>=5.1.0",
+    "tomli>=1.2.2, <2.0.0"
```

Comment:
let's chat about this

Reply:
sure!

Reply:
I meant a bit more detailed description, fe: `conflict caused by "black", waiting for a new version release`

Reply:
ah fair 😅 , will just take over your example then